pages/linux/warpd.md

```diff
@@ -0,0 +1,32 @@
+# warpd
+
+> A modal keyboard driven pointer manipulation program.
+> More information: <https://github.com/rvaiya/warpd/blob/master/man.md>.
+
+- Run warpd in normal mode:
+
+`warpd --normal`
+
+- Move cursor left:
+
+`h`
+
+- Move cursor down:
+
+`j`
+
+- Move cursor up:
+
+`k`
+
+- Move cursor right:
+
+`l`
+
+- Emulate left click:
+
+`m`
+
+- Run warpd in hint mode:
+
+`warpd --hint`
```

Comment:
We use to list examples about command arguments, then hot keys for interactive mode. So, IMO this should be the 2nd example in this page.

Reply:
Agreed that was my suggestion too, as it would better align with our other pages. The PRs Author commented on this above.